src/ert/ensemble_evaluator/_builder/_legacy.py

```diff
@@ -39,6 +39,7 @@
 
 logger = logging.getLogger(__name__)
 event_logger = logging.getLogger("ert.event_log")
+job_queue_or_scheduler_logger = logging.getLogger("ert.job_queue_or_scheduler_log")
```

Comment:
I still don't like the name tbh. What about just scheduler_logger?

Reply:
We already have a ert.scheduler logger. Do you think I could reuse that one?

Reply:
I interpreted the issue as "make a specific logger only for this purpose", but I do not think it matters that we re-use the scheduler one.